src/imp/tracer_imp.js

```diff
@@ -1187,7 +1200,7 @@ export default class Tracer extends opentracing.Tracer {
 
                     if (res.commandsList && res.commandsList.length > 0) {
                         if (res.commandsList[0].devMode) {
-                            this.options().meta_event_reporting = true;
+                            this.options().disable_meta_event_reporting = false;
```

Comment:
Here we should set another field to the value of `!disable_meta_event_reporting && res.commandList[0].devMode`.

Reply:
Also as a small measure of future-proofing, should we not assume `commandsList[0]` is the only command? We've talked about potentially having a command to tell the tracer to issue a log. We've potentially talked about a Disable command, all of this could happen relatively soon, would be good not to hard-code this 0 here.

Reply:
`commands` is an array with either zero or one object in it, and all commands are in that single object. (we get `[{devMode:false, disabled:false}]` for example)